ios/Video/RCTVideo.m

```diff
@@ -438,13 +433,28 @@ - (void)playerItemPrepareText:(AVAsset *)asset assetOptions:(NSDictionary * __nu
                            atTime:kCMTimeZero
                             error:nil];
   
-  AVAssetTrack *audioAsset = [asset tracksWithMediaType:AVMediaTypeAudio].firstObject;
-  AVMutableCompositionTrack *audioCompTrack = [mixComposition addMutableTrackWithMediaType:AVMediaTypeAudio preferredTrackID:kCMPersistentTrackID_Invalid];
-  [audioCompTrack insertTimeRange:CMTimeRangeMake(kCMTimeZero, videoAsset.timeRange.duration)
-                          ofTrack:audioAsset
-                           atTime:kCMTimeZero
-                            error:nil];
-  
+  // load alternative audio source, if provided
+  if (audio != nil) {
```

Comment:
Maybe I am half blind, but I don't see where this audio var comes from

Reply:
oh yea :D sry, added that now